.github/workflows/merge-to-master.yml

```diff
@@ -8,13 +8,16 @@ on:
 jobs:
   cd:
     name: Continuous Delivery pipeline
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
 
     steps:
-      - name: Set up Go 1.14
+      - name: Set up Go ^1.20
```

Comment:
This was the version being used... 1.20.3 but it's hidden behind ^1.14, updating it to something more according and also adding a go version step to ensure what version is running.